schema/src/main/scala/io/circe/schema/FormatValidator.scala

```diff
@@ -0,0 +1,9 @@
+package io.circe.schema
+
+import org.everit.json.schema.{ FormatValidator => EveritFormatValidator }
+
+trait FormatValidator extends EveritFormatValidator {
+  def name: String
```

Comment:
I couldn't come up with a different way of enforcing `formatName` implementation, since `formatName` has a default value `"unnamed-format"`